scala-test-suite/src/test/resources/2.13.3/BlacklistedTests.txt

```diff
@@ -0,0 +1,218 @@
+scala/tools/nsc/transform/MixinTest.scala
+scala/tools/nsc/transform/ReleaseFenceTest.scala
+scala/tools/nsc/transform/SpecializationTest.scala
+scala/tools/nsc/transform/ThicketTransformerTest.scala
+scala/tools/nsc/transform/UncurryTest.scala
+scala/tools/nsc/transform/delambdafy/DelambdafyTest.scala
+scala/tools/nsc/transform/patmat/SolvingTest.scala
+scala/tools/nsc/transform/patmat/PatmatBytecodeTest.scala
+scala/tools/nsc/typechecker/ImplicitsTest.scala
+scala/tools/nsc/typechecker/InferencerTest.scala
+scala/tools/nsc/typechecker/NamerTest.scala
+scala/tools/nsc/typechecker/OverridingPairsTest.scala
+scala/tools/nsc/typechecker/ParamAliasTest.scala
+scala/tools/nsc/typechecker/TypedTreeTest.scala
+scala/tools/nsc/util/StackTraceTest.scala
+scala/util/ChainingOpsTest.scala
+
+## Do not link
+scala/CollectTest.scala
+scala/MatchErrorSerializationTest.scala
+scala/PartialFunctionSerializationTest.scala
+scala/lang/stringinterpol/StringContextTest.scala
+scala/collection/IterableTest.scala
+scala/collection/IteratorTest.scala
+scala/collection/NewBuilderTest.scala
+scala/collection/SeqViewTest.scala
+scala/collection/SetMapConsistencyTest.scala
+scala/collection/SetMapRulesTest.scala
+scala/collection/Sizes.scala
+scala/collection/concurrent/TrieMapTest.scala
+scala/collection/convert/WrapperSerializationTest.scala
+scala/collection/immutable/ChampMapSmokeTest.scala
+scala/collection/immutable/ChampSetSmokeTest.scala
+scala/collection/immutable/LazyListGCTest.scala
+scala/collection/immutable/LazyListLazinessTest.scala
+scala/collection/immutable/ListTest.scala
+scala/collection/immutable/SerializationTest.scala
+scala/collection/immutable/StreamTest.scala
+scala/collection/immutable/StringLikeTest.scala
+scala/collection/immutable/VectorTest.scala
+scala/collection/mutable/AnyRefMapTest.scala
+scala/collection/mutable/ArrayBufferTest.scala
+scala/collection/mutable/OpenHashMapTest.scala
+scala/collection/mutable/PriorityQueueTest.scala
+scala/collection/mutable/SerializationTest.scala
+scala/concurrent/FutureTest.scala
+scala/concurrent/duration/SerializationTest.scala
+scala/concurrent/impl/DefaultPromiseTest.scala
+scala/io/SourceTest.scala
+scala/jdk/AccumulatorTest.scala
+scala/jdk/DurationConvertersTest.scala
+scala/jdk/FunctionConvertersTest.scala
+scala/jdk/OptionConvertersTest.scala
+scala/jdk/StepperConversionTest.scala
+scala/jdk/StepperTest.scala
+scala/jdk/StreamConvertersTest.scala
+scala/jdk/StreamConvertersTypingTest.scala
+scala/math/OrderingTest.scala
+scala/runtime/ScalaRunTimeTest.scala
+scala/sys/env.scala
+scala/sys/process/ParserTest.scala
+scala/sys/process/PipedProcessTest.scala
+scala/sys/process/ProcessBuilderTest.scala
+scala/sys/process/ProcessTest.scala
+scala/tools/testkit/AssertUtilTest.scala
+scala/util/PropertiesTest.scala
+scala/util/SpecVersionTest.scala
+scala/util/SystemPropertiesTest.scala
+
+# Do not link because they use AssertUtil whose constructor refers to java.lang.Field
```

Comment:
This is new. `AssertUtil` receive a `private lazy val` field of type `java.lang.Field`, which pollutes all of `AssertUtil` for Scala.js. I intend to submit a change upstream to recover those tests in the next version.

Reply:
Maybe file an issue to make sure we don't forget to remove these when upgrading to 2.13.4?

Reply:
#4161